packages/flutter_tools/templates/module/ios/library/Flutter.tmpl/podhelper.rb.tmpl

```diff
@@ -15,6 +15,21 @@ require 'json'
 #                                          Optional, defaults to two levels up from the directory of this script.
 #                                          MyApp/my_flutter/.ios/Flutter/../..
 def install_all_flutter_pods(flutter_application_path = nil)
+  # defined_in_file is a Pathname to the Podfile set by CocoaPods.
+  pod_contents = File.read(defined_in_file)
+  unless pod_contents.include? 'flutter_post_install'
+    puts  <<~POSTINSTALL
+Add `flutter_post_install(installer)` to your Podfile `post_install` block to build Flutter plugins:
+
+post_install do |installer|
+  flutter_post_install(installer)
+end
+POSTINSTALL
+    raise 'Missing `flutter_post_install(installer)` in Podfile `post_install` block'
+  end
+
+  require File.expand_path(File.join('packages', 'flutter_tools', 'bin', 'podhelper'), flutter_root)
```

Comment:
nit, would be nice if this were lifted to the top. however, would require a refactor, as the `flutter_root` function can't be called until it's defined, because in ruby function definitions are expression statements evaluated in order. so not a blocker for this PR.